### ISSUE TITLE
Claiming namespace [bmcsoftware]

### ISSUE THREAD

**Issue by Orch3strator** [EclipseFdn/open-vsx.org#926]

**Comment by kineticsquid** [comment]
@Orch3strator can you provide background on what connection you have to this workspace? 

FYI, the repo link, https://github.com/controlm/automation-api-community-solutions/6-ide-integrations/integration-via-marketplace/vscode,  on your extension page results in a 404.

**Comment by Orch3strator** [comment]
Hello,

I've updated the git repo as we [BMC Software] are trying to figure out how to properly open-source on our Control-M GitHub. The current repo is located at [TryBMC GitLab](https://ctm-git.trybmc.com/community/vscode-ctm), and will move back to our ControlM repo after we've cleaned up the structure. However, I don't want to wait until we get this done. 

I'm working for BMC Software and created the code for the app. My personal GitHub account is associated with my work for BMC Software. 

Hope that helps.

Regards, V.
